docker/Dockerfile

```diff
@@ -2,21 +2,10 @@
 # docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile
 FROM continuumio/miniconda3
 
-# update the conda packages
-RUN conda update -y conda pip
-
-
-# install environment packages
-COPY . /home/root/source
-WORKDIR /home/root/source
-RUN ls
-RUN conda env update --name base --file environment.yml
-RUN pip install .
-RUN conda clean --all -y
-RUN rm -r /home/root/source
```

Comment:
Would it still make sense to remove the source code directory after installation to reduce the size?

Reply:
I am not sure this is having a great impact as Docke stores the containers by layers and the source is in the previous layers. 

What I have seen is critical is reducing the number of layers and do the cleaning at the same time.